src/GraphQL/ResolverRegistryInterface.php

```diff
@@ -50,6 +51,9 @@ public function addFieldResolver($type, $field, ResolverInterface $resolver);
   /**
    * Return the field resolver for a given type and field name.
    *
+   * Most code will probably want to use `getFieldResolverWithInheritance`
+   * instead.
```

Comment:
Not sure if we should keep this comment on the interface?

" ... which does not exist on this interface yet but on the ResolverRegistry class"

Reply:
Good point. I think in 5.0.0 we may even want to just swap around the implementation of `getFieldResolver` 🤔 

I think the biggest reason I didn't just implement the functionality there in the first place was that I wasn't sure if that'd break anything for users expecting a certain behaviour. Getting the implementation with inheritance should probably be the default behaviour.